packages/components/src/components/contract-card/contract-card-items/turbos-card-body.jsx

```diff
@@ -0,0 +1,153 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { isCryptocurrency, getLimitOrderAmount, getTotalProfit, isValidToSell } from '@deriv/shared';
+import ContractCardItem from './contract-card-item.jsx';
+import ToggleCardDialog from './toggle-card-dialog.jsx';
+import Icon from '../../icon';
+import MobileWrapper from '../../mobile-wrapper';
+import Money from '../../money';
+import { ResultStatusIcon } from '../result-overlay/result-overlay.jsx';
+
+const TurbosCardBody = ({
+    addToast,
+    connectWithContractUpdate,
+    contract_info,
+    contract_update,
+    currency,
+    current_focus,
+    error_message_alignment,
+    getCardLabels,
+    getContractById,
+    is_sold,
+    onMouseLeave,
+    removeToast,
+    setCurrentFocus,
+    status,
+    is_mobile,
+    progress_slider_mobile_el,
+}) => {
+    const total_profit = getTotalProfit(contract_info);
+    const { buy_price, profit, limit_order, barrier, current_spot_display_value, sell_spot, entry_spot } =
+        contract_info;
+    const { take_profit } = getLimitOrderAmount(contract_update || limit_order);
+    const is_valid_to_sell = isValidToSell(contract_info);
+    const { BARRIER_LEVEL, BUY_PRICE, CURRENT_PRICE, STAKE, TAKE_PROFIT, TOTAL_PROFIT_LOSS, PAYOUT, PROFIT_LOSS } =
+        getCardLabels();
+
+    return (
+        <React.Fragment>
+            <div className={is_mobile ? 'dc-contract-card-items-wrapper--mobile' : 'dc-contract-card-items-wrapper'}>
+                <ContractCardItem
+                    className={is_mobile ? 'dc-contract-card__stake-mobile' : 'dc-contract-card__stake'}
```

Comment:
```suggestion
                    className='dc-contract-card__stake'
```
@kate-deriv one more suggestion😊 
we can avoid adding is_mobile ? 'dc-contract-card__stake-mobile' : to every child
instead, we can nest them in scss.
First we open packages/components/src/components/contract-card/contract-card.scss line **276** where - dc-contract-card-items-wrapper--mobile is declared and nest children inside of it:
```
&--mobile {
     // styles for dc-contract-card-items-wrapper--mobile
     
     .dc-contract-card__stake {
     // styles for dc-contract-card__stake which is a child of dc-contract-card-items-wrapper--mobile
     // it will be applied when is_mobile is true 
```
so we won't need to add `--mobile` suffix to each child🙂

Reply:
Done, thank you